Geometry/HcalCommonData/BuildFile.xml

```diff
@@ -1,6 +1,7 @@
 <use   name="DetectorDescription/Base"/>
 <use   name="DetectorDescription/Core"/>
 <use   name="DataFormats/HcalDetId"/>
+<use   name="SimDataFormats/CaloTest"/>
```

Comment:
@bsunanda I would prefer to resolve things here. If we can remove HcalTestHistoClass, it will be easy to move HcalTestNumbering to Geometry/HcalCommonData.